src/app/routes/profile/containers/editprofile.js

```diff
@@ -219,147 +220,129 @@ class EditProfile extends React.Component {
         }
 
         return(
-            <div id="content">
-                <br/>
-                <div className="row">
-                    <article className="col-sm-10 col-sm-offset-1 col-md-10 col-md-offset-1 col-lg-8 col-lg-offset-2">
-                        <JarvisWidget editbutton={false} custombutton={false}>
-                            <header>
-                                <span className="widget-icon"> <i className="fa fa-edit"/> </span>
-                                <h2>Edit Profile</h2>
-                            </header>
-                            <div>
-                                <div className="widget-body no-padding">
-                                    <UiValidate options={this.validationOptions}>
-                                        <form id="checkout-form" className="smart-form" noValidate="novalidate">
-                                            <fieldset>
-                                                <div className="row">
-                                                    <div className="text-center">
-                                                        <div className="">
-                                                            <img className="img-thumbnail" name="image" ref="image" style={{objectFit: 'cover', height: "120px", width: "120px"}} src={this.state.photoURL}/><br/>
-                                                        </div>
-                                                        <label className="btn btn-link">Select Photo
-                                                            <input type="file" name="imageSelect" ref="imageSelect" style={{display: 'none'}} onChange={this.photoHandler}/>
-                                                        </label><br/><br/>
-                                                    </div>
+            <div id="content" className="container-fluid animated fadeInDown">
+                <h3 className="text-center text-danger">Edit Profile</h3><hr/>
+                    <div className="col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1" style={boxShadowStyle}>
+                        <UiValidate options={this.validationOptions}>
+                            <form id="checkout-form" className="smart-form" noValidate="novalidate">
+                                <fieldset>
+                                    <div className="row">
+                                        <div className="col-lg-4 col-md-4">
+                                            <div className="text-center">
+                                                <div className="">
+                                                    <img className="img-thumbnail" name="image" ref="image" style={bigCirclePhotoStyle} src={this.state.photoURL}/><br/>
```

Comment:
should this be 'this.profile.photoURL' as well as for the following places where you try to access the profile data from the local state?